### PR TITLE
Es6

### PR DESCRIPTION
**Pull request by anthony-redFox**

### CODE REVIEW

**Comment by anthony-redFox** [comment]
The next PR will remove lodash dependency

File: lib/preprocessor.js [hunk 1]
```diff
@@ -80,35 +79,36 @@ function createCoveragePreprocessor (logger, helper, basePath, reporters, covera
...
-        return false
       }
-      return acc
-    }, true)
+      return notIncluded
+    })
   }
 
-  if (!checkInstrumenters()) {
+  if (checkInstrumenters()) {
```
Comment (johnjbarton) [inline lib/preprocessor.js]: How can dropping `!` be ok here?

File: lib/preprocessor.js [hunk 2]
```diff
@@ -118,14 +118,13 @@ function createCoveragePreprocessor (logger, helper, basePath, reporters, covera
     log.debug('Processing "%s".', file.originalPath)
 
     var jsPath = path.resolve(file.originalPath)
```
Comment (johnjbarton) [inline lib/preprocessor.js]: No var in es6

File: lib/preprocessor.js (hunk 1 above)
Comment (anthony-redFox) [reply lib/preprocessor.js]: checkInstrumenters also changed that mean checkInstrumenters return true if we don't find instrument

**Comment by johnjbarton** [comment]
Thanks!

**Pull request merged**